### PR TITLE
Ticket5270 fmr

### PR DESCRIPTION
**Pull request by davidkeymer**
Support module containing LVDCOM configuration files.

### CODE REVIEW

**Comment by davidkeymer** [comment]
### Notes for Reviewer
- Source VI included for reference only as it's not executable; it doesn't contain the necessary drivers for the VNA nor the libraries for the Hall probe.
- Versions for LabVIEW 2010 and 2017 placed in `protocol` directory.  Whichever is chosen to be used for the review will need to be renamed `VNA-FMR-POLREF.vi` to match that in the LVDCOM configuration file `lv_FMR.xml`.

File: FMRApp/protocol/lv_FMR.xml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+<!--Initially generated by $Id$ -->
+<lvinput xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://epics.isis.rl.ac.uk/lvDCOMinput/1.0" xsi:schemaLocation="http://epics.isis.rl.ac.uk/lvDCOMinput/1.0 lvDCOMinput.xsd">
+  <!--Full path to external interface vi used for triggering LabVIEW events when extint="true" below -->
+  <extint path="$(LVDCOM=$(TOP))/lvDCOMApp/src/extint/Main/Library/External Interface - Set Value.vi">
+    <!--Look for extint in LVDCOM directory or TOP if not defined-->
+  </extint>
+  <!--Definition of a configSection, the second argument of the IOC lvDCOMConfigure() command -->
+  <section name="frontpanel">
+    <!--TODO: edit path attribute in <vi /> to contain a full absolute path, you can also use any EPICS macro known to the IOC e.g. $(TOP) or $(LVDCOM)-->
+    <vi path="C:/Users/wil0322903/Desktop/VNA-FMR/VNA-FMR-POLREF.vi">
```
Comment (DominicOram) [inline FMRApp/protocol/lv_FMR.xml]: This path isn't right. Can you point it at the VI within this folder? Or if it's going to change make it a macro.

**Review by DominicOram — CHANGES_REQUESTED** [review]
Looks like it would work if path is corrected.

File: FMRApp/protocol/lv_FMR.xml (hunk 1 above)
Comment (davidkeymer) [reply FMRApp/protocol/lv_FMR.xml]: Macro added.

**Review by DominicOram — APPROVED** [review]
Works great and conforms much better to standards, thank you. Only issue is that specifying a `EGU` for binary and string records doesn't make any sense so you get errors like `ErrorRecord "TE:NDLT1171:FMR_01:VNA:STATE:SP" does not have a field "EGU"` when you start the IOC, I will correct this for you.

**Pull request merged**